pkgs/by-name/op/opensmtpd-table-redis/package.nix

```
@@ -43,7 +43,7 @@ stdenv.mkDerivation (finalAttrs: {
 
   meta = {
     homepage = "https://www.opensmtpd.org/";
-    description = "ldap table for the OpenSMTPD mail server";
+    description = "redis tables for the OpenSMTPD mail server";
```

```suggestion
    description = "Redis tables for the OpenSMTPD mail server";
```

To make it consistent with all the other cases.